### PR TITLE
Christian/c 6951 create overall footer component in

### PR DESCRIPTION
**Pull request by heyglassy**
## Description

Consolidated Business and free tier footers into one footer.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/react-hooks/src/preferences/types.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export interface PreferenceState {
   preferences?: IPreferenceTemplate[];
   recipientPreferences?: IRecipientPreference[];
   preferenceSections?: Array<any>;
+  preferencePage?: any;
```
Comment (suhasdeshpande) [inline packages/react-hooks/src/preferences/types.ts:7]: do we need preferenceSections and preference page at the same level?

Comment (heyglassy) [reply packages/react-hooks/src/preferences/types.ts:7]: for now, I'll create another ticket to move deprecate preferenceSections.

File: packages/react-preferences/src/components/BusinessFooter.tsx [hunk 2]
```diff
@@ -31,46 +31,49 @@ const FooterWrapper = styled.div`
...
+}) => {
   return (
     <FooterWrapper>
       <SVGWrapper>
-        <Facbebook />
-        <Instagram />
-        <LinkedIn />
-        <Medium />
-        <Twitter />
+        {links?.facebook?.url && (
```
Comment (suhasdeshpande) [inline packages/react-preferences/src/components/BusinessFooter.tsx:52]: This feels a bit awkward datastrucuture-wise. might be easier to map this to a  strongly typed collection on the API side.

```ts
type SocialLinks = Array<{type: "facebook" | "Instagram" | string, value: string}>
```

File: packages/storybook/stories/inbox/examples.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+            brand={{
+              colors: {
+                primary: "red",
+                secondary: "pink",
+                tertiary: "orange",
+              },
+            }}
+          />
+        </CourierProvider>
+      </div> */}
```
Comment (suhasdeshpande) [inline packages/storybook/stories/inbox/examples.stories.tsx:198]: do we need this?

Comment (heyglassy) [reply packages/storybook/stories/inbox/examples.stories.tsx]: fixed

**Review by suhasdeshpande — APPROVED** [review]

**Pull request merged**